src/mappers.ts

```diff
@@ -173,7 +164,7 @@ export function getResourceId() {
 }
 function validateName(value:string) 
 {
-    var r = new RegExp(/[^a-zA-Z0-9_-]/);
+    var r = new RegExp(/[^a-zA-Z0-9_. -]/);
```

Comment:
We don't want to allow . as it is not a valid character for test name. The API will give an error with . Can you change this?